packages/components/src/InlineMessage/index.js

```diff
@@ -0,0 +1 @@
+export * from './InlineMessage.component';
```

Comment:
You should export it to index.js at root level too

Reply:
Yep, I added import to component's index file

Reply:
https://github.com/Talend/ui/pull/2837/files#diff-5024897ca9c78ec120fc210faa8ab3adR188